content/sensu-go/6.1/observability-pipeline/observe-process/handlers.md

```diff
@@ -44,13 +44,72 @@ TCP and UDP handlers enable Sensu to forward event data to arbitrary TCP or UDP
 
 ## Handler sets
 
-Handler set definitions allow you to use a single named handler set to refer to groups of handlers (individual collections of actions to take on event data).
+Handler set definitions allow you to use a single named handler set to refer to groups of handlers.
+The handler set becomes a collection of individual actions to take (via each included handler) on event data.
+
+For example, suppose you have already created these two handlers:
+
+- `elasticsearch` to send all observation data to Elasticsearch.
+- `opsgenie` to send non-OK status alerts to your OpsGenie notification channel.
+
+You can list both of these handlers in a handler set to automate and streamline your workflow, specifying `type: set`:
+
+{{< language-toggle >}}
+
+{{< code yml >}}
+type: Handler
+api_version: core/v2
+metadata:
+  name: send_events_notify_operator
+  namespace: default
+spec:
+  handlers:
+  - elasticsearch
+  - opsgenie
+  type: set
+{{< /code >}}
+
+{{< code json >}}
+{
+  "type": "Handler",
+  "api_version": "core/v2",
+  "metadata": {
+    "name": "send_events_notify_operator",
+    "namespace": "default"
+  },
+  "spec": {
+    "type": "set",
+    "handlers": [
+      "elasticsearch",
+      "opsgenie"
+    ]
+  }
+}
+{{< /code >}}
+
+{{< /language-toggle >}}
+
+Now you can route observation data to Elasticsearch and alerts to OpsGenie with a single handler, the `send_events_notify_operator` handler set.
 
 {{% notice note %}}
-**NOTE**: Attributes defined on handler sets do not apply to the handlers they include.
+**NOTE**: Attributes defined in handler sets do not apply to the handlers they include.
 For example, `filters` and `mutator` attributes defined in a handler set will have no effect on handlers.
+Define these attributes in individual handlers instead.
 {{% /notice %}}
 
+## Handler stacks
+
+Handler stacks are handler sets that escalate events through a series of different handlers.
+For example, consider a handler stack with three levels of escalation:
+
+- Level 1: On first occurrence, attempt remediation with the `remediation` handler.
+- Level 2: On fifth occurrence, send an alert to Slack with the `slack_alert_channel` handler.
+- Level 3: On tenth occurrence, send an alert to PagerDuty with the `pagerduty_emergency` handler.
+
+The escalation levels are controlled by event filter definitions.
+At level 1, no filter is needed because every event triggers remediation with the `remediation` handler.
```

Comment:
No filter needed is not entirely correct.  You would still want the `is_incident` filter otherwise on the initial successful check the remediation would still be ran.